<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Add_Issue</class>
 <widget class="QDialog" name="Dialog_Add_Issue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加/修改事件</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>49</y>
     <width>371</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="StandardLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelofTitle">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>*Title:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QTextEdit" name="Titletext">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelofDescription">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QTextEdit" name="Descriptiontext">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>61</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelofPosition">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Position:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QTextEdit" name="Positiontext">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelofStartTime">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>*StartTime:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QDateTimeEdit" name="StartTimeedit">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="Alldaycheck">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Allday</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelofIssueType">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>*IssueType</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QComboBox" name="IssueTypeBox">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelofRepeat">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Repeat</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelofMei">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>每</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="numberBox">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QComboBox" name="RepeatTypeBox">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ButtonLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="CancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="AcceptButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>232</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="EndTimeLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelofEndTime">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>*EndTime:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="EndTimeedit">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
